$OpenBSD: patch-configure_ac,v 1.2 2014/06/04 17:40:27 jasper Exp $

Enable libffi on hppa, m88k, mips64(el), powerpc and vax platforms.

--- configure.ac.orig	Sun Mar 17 23:36:15 2013
+++ configure.ac	Sun Apr 20 18:16:16 2014
@@ -191,6 +191,10 @@ case "$host" in
 	TARGET=M68K; TARGETDIR=m68k
 	;;
 
+  m88k-*-*)
+	TARGET=M88K; TARGETDIR=m88k
+	;;
+
   microblaze*-*-*)
 	TARGET=MICROBLAZE; TARGETDIR=microblaze
 	;;
@@ -256,6 +260,10 @@ case "$host" in
         TARGET=TILE; TARGETDIR=tile
         ;;
 
+  vax-*-openbsd*)
+	TARGET=VAX; TARGETDIR=vax
+	;;
+
   xtensa*-*)
 	TARGET=XTENSA; TARGETDIR=xtensa
 	;;
@@ -281,6 +289,7 @@ AM_CONDITIONAL(ALPHA, test x$TARGET = xALPHA)
 AM_CONDITIONAL(IA64, test x$TARGET = xIA64)
 AM_CONDITIONAL(M32R, test x$TARGET = xM32R)
 AM_CONDITIONAL(M68K, test x$TARGET = xM68K)
+AM_CONDITIONAL(M88K, test x$TARGET = xM88K)
 AM_CONDITIONAL(MICROBLAZE, test x$TARGET = xMICROBLAZE)
 AM_CONDITIONAL(METAG, test x$TARGET = xMETAG)
 AM_CONDITIONAL(MOXIE, test x$TARGET = xMOXIE)
@@ -302,6 +311,7 @@ AM_CONDITIONAL(PA_HPUX, test x$TARGET = xPA_HPUX)
 AM_CONDITIONAL(PA64_HPUX, test x$TARGET = xPA64_HPUX)
 AM_CONDITIONAL(TILE, test x$TARGET = xTILE)
 AM_CONDITIONAL(XTENSA, test x$TARGET = xXTENSA)
+AM_CONDITIONAL(VAX, test x$TARGET = xVAX)
 
 AC_HEADER_STDC
 AC_CHECK_FUNCS(memcpy)
